src/main/java/uk/gov/hmcts/reform/sendletter/logging/AppInsights.java

```diff
@@ -24,27 +23,28 @@
 
     static final DateTimeFormatter TIME_FORMAT = DateTimeFormatter.ofPattern("HH:mm:ss");
 
-    public AppInsights(TelemetryClient telemetry) {
-        super(telemetry);
+    private final TelemetryClient telemetry;
 
+    public AppInsights(TelemetryClient telemetry) {
+        this.telemetry = telemetry;
     }
 
     // dependencies
 
-    private void trackDependency(String dependency, String command, java.time.Duration duration, boolean success) {
-        telemetry.trackDependency(dependency, command, new Duration(duration.toMillis()), success);
+    private void trackDependency(String command, java.time.Duration duration, boolean success) {
+        telemetry.trackDependency(AppDependency.FTP_CLIENT, command, new Duration(duration.toMillis()), success);
     }
 
     public void trackFtpUpload(java.time.Duration duration, boolean success) {
-        trackDependency(AppDependency.FTP_CLIENT, AppDependencyCommand.FTP_FILE_UPLOADED, duration, success);
+        trackDependency(AppDependencyCommand.FTP_FILE_UPLOADED, duration, success);
     }
 
     public void trackFtpReportDeletion(java.time.Duration duration, boolean success) {
-        trackDependency(AppDependency.FTP_CLIENT, AppDependencyCommand.FTP_REPORT_DELETED, duration, success);
+        trackDependency(AppDependencyCommand.FTP_REPORT_DELETED, duration, success);
     }
 
     public void trackFtpReportDownload(java.time.Duration duration, boolean success) {
-        trackDependency(AppDependency.FTP_CLIENT, AppDependencyCommand.FTP_REPORT_DOWNLOADED, duration, success);
+        trackDependency(AppDependencyCommand.FTP_REPORT_DOWNLOADED, duration, success);
```

Comment:
are those changes related to library upgrade?

Reply:
Please forgive for not outlining this in the description. As part of the upgrade one should notice there's a small boy scout task available in the same file. Absolutely harmless one might add.

If it is still up to a challenge for a new PR - by all means I will revert this change